book/website/profiles.yml

```diff
@@ -107,9 +118,56 @@
     research software. This curated set of chapters guides the sharing and
     citation of software components from source code to computational workflows
     and executables while attributing all contributors fairly.
-- name: Multinational Collaboration
+- name: Data Science Without Borders
   files:
+    - reproducible-research/reproducible-research
+    - reproducible-research/overview/overview-definitions
+    - reproducible-research/vcs
+    - reproducible-research/vcs/vcs-checklist
+    - reproducible-research/open
+    - reproducible-research/open/open-source
+    - reproducible-research/open/open-data
+    - reproducible-research/open/open-hardware
+    - reproducible-research/open/open-access
+    - reproducible-research/open/open-scholarship
+    - reproducible-research/licensing
+    - reproducible-research/rdm
+    - reproducible-research/rdm/rdm-dmp
+    - reproducible-research/rdm/rdm-fair
+    - reproducible-research/rdm/rdm-metadata
+    - reproducible-research/rdm/rdm-spreadsheets
+    - reproducible-research/rdm/rdm-data-curation
+    - reproducible-research/rdm/rdm-sharing
+    - reproducible-research/rdm/rdm-checklist
+    - reproducible-research/code-documentation
+    - reproducible-research/testing
+    - reproducible-research/testing/testing-overview
+    - reproducible-research/testing/testing-checklist
+    - reproducible-research/code-quality
+    - reproducible-research/code-quality/code-quality-style
+    - reproducible-research/reviewing/reviewing-workflow
+    - reproducible-research/reviewing/reviewing-checklist
+    - reproducible-research/code-reuse/code-reuse-details
+    - project-design/project-design
+    - project-design/pd-checklist
+    - project-design/project-repo
+    - project-design/filenaming
+    - project-design/sdp
+    - project-design/sdp/metadata
+    - project-design/sdpm
+    - project-design/sdpw
+    - project-design/data-governance
+    - project-design/missing-data
+    - project-design/risks-of-bias
+    - communication/communication
     - communication/comms-overview
+    - communication/os-comms
+    - communication/dif-articles
+    - communication/presentations
+    - communication/social-media
```

Comment:
This is great, but we can remove it for now as it might not be needed for the DSWB chapter, currently.

Reply:
Yes! Please drop social media chapter as it is about X and we don't want to promote that.